tests/testthat/test-save-separate-chunks.R

```diff
@@ -0,0 +1,72 @@
+context("save chunks")
+
+library(cdcfluview)
+# retrieve state-level data from the CDC's FluView Portal
+state_flu <- get_state_data(2008:2014)
+# data clean
```

Comment:
Rather than downloading the data during each run of this test, please just download it once and store it as a list called `FluView` that contains all the data that you need for this data viz. Then do `save(FluView, file="animint/data/FluView.RData")` and in this test do

```r
data(FluView)
```